src/main/react-front-end/src/views/Transfer/EndpointBrowseComponent.js

```diff
@@ -283,8 +281,18 @@ export default class EndpointBrowseComponent extends Component {
 			this.setState({directoryPath: path, ids: id});
 			setLoading(false);
 		}, (error) =>{
-			this._handleError(error);
+			this._handleError("Login Failed. Re-directing to OAuth page");
```

Comment:
In my opinion, 'Login Failed' is not a valid message. Rather we need to say something like 'Saved authentication details have expired. You will need to reauthenticate this account'. I feel it is possible to read this in 2 secs.